enhancements/scheduling/prevent-user-workloads-from-being-scheduled-on-control-plane-nodes.md

```diff
@@ -306,36 +399,34 @@ Tainting a single-node cluster would not make sense, then this workflow is not a
 
 ### Implementation Details/Notes/Constraints
 
-See Alternatives section. The previous discarded approach required actual implementation of a admission plugin/controller or some other way to avoid tolerations with new code provided by us, but the accepted workflow is now available out-of-the-box with ValidatingAdmissionPolicies with CEL (still in tech preview, but available if enabled) combined with the NoExecute taint enforcement. So no implementation from our side is needed, only documentation and guidance on how to apply this workflow.
+See Alternatives section. The previous discarded approach required actual implementation of a admission plugin/controller or some other way to avoid tolerations with new code provided by us, but the accepted workflow is now available out-of-the-box with ValidatingAdmissionPolicies with CEL (still in tech preview, but available if enabled) combined with the NoExecute taint enforcement. So no real implementation from our side is needed, besides providing the default ValidatingAdmissionPolicy and writing documentation/guidance on how to apply this workflow.
```

Comment:
Are owners of OpenShift control-plane workloads expected to change anything? E.g. label namespaces (the VAP still expresses `openshift.io/control-plane` in its namespaceSelector.matchExpression) and make sure every OpenShift workload landing on either of the control-plane nodes tolerate the new taint?

s/our/OpenShift control plan components

Reply:
That's left over. I will change it. No labeling of namespaces should be needed now. Only rbac. 

I'll update this text to be more clear with requirements from openshift Devs. 

We need to provide the VAP,  tolerate the right taints, and update rbac to actually be able to tolerate those taints in case the vap is binded